app/helpers/application_helper/toolbar/miq_policy_center.rb

```diff
@@ -67,7 +67,7 @@ class ApplicationHelper::Toolbar::MiqPolicyCenter < ApplicationHelper::Toolbar::
           :options   => {:feature => 'miq_policy_edit_events'}
         ),
         button(
-          :miq_event_edit,
+          :miq_policy_event_edit,
```

Comment:
Fixes a bug for non-admin users where this screen would fail to load this toolbar... fixed by @kavyanekkalapu 

before...

![image](https://user-images.githubusercontent.com/19339/119039823-ae887e80-b982-11eb-8f88-d703c1b3b833.png)


after:


![image](https://user-images.githubusercontent.com/19339/119039635-7b45ef80-b982-11eb-9447-ee50e2cee2e9.png)